libs/core/src/lib/info-label/info-label.component.ts

```diff
@@ -0,0 +1,54 @@
+import { Component, ViewEncapsulation, ChangeDetectionStrategy, Input, OnInit, ElementRef } from '@angular/core';
+
+type LabelType = 'numeric' | 'only-icon' | 'icon';
+
+@Component({
+    selector: 'fd-info-label',
+    templateUrl: './info-label.component.html',
+    styleUrls: ['./info-label.component.scss'],
+    encapsulation: ViewEncapsulation.None,
+    changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class InfoLabelComponent implements OnInit {
+
+    /** Select type defines the label types */
+    @Input()
+    labelType: LabelType;
+
+    /** define the icon type */
+    @Input()
+    glyph: String; 
```

Comment:
Let's change `String` to lowercase `string`. changes are easier to detect on primitive types